### PR TITLE
test: create unit tests for preprocess and merge functions

### LINKED ISSUE #5: Add unit tests for preprocess and merging functions.
Add test units for preprocessing and merge functions.

### PR DESCRIPTION
**Pull request by welozano**
Create test units for preprocessing and merge functions. The code was refactored to facilitate the implementation of the test units. Closes #5

### CODE REVIEW

File: src/gt_merger/args.py [hunk 1]
```diff
@@ -28,20 +29,21 @@ def get_parser():
 
     parser.add_argument('--gtFile', type=str, required=True, help='Path to XLSX file including the Ground Truth data')
 
-    parser.add_argument('--dataDir', type=str, default='data',
-                        help='Folder including the sub-folder related to each GT Data files')
+    parser.add_argument('--dataDir', type=str, default=constants.DATA_DIR,
+                        help='Folder used to save output and log data from preprocess and merge')
```
Comment (barbeau) [inline src/gt_merger/args.py]: Maybe `dataDir` should be renamed to `outputDir`? Reading this description, it wasn't immediately obvious to me if this was an input folder that this script consumes data from or output folder where the result of this script is outputting data. The description in the README is better, but this should still contain a good explanation of what this parameter is.

**Review by barbeau — COMMENTED** [review]
Thanks @welozano, this looks good! One comment in-line.

File: src/gt_merger/args.py (hunk 1 above)
Comment (welozano) [reply src/gt_merger/args.py]: You are right. Initially dataDir was intended to holds the input files (assuming there were more than two). After adding the parameters for input files (oba and GT), then I decided to use it to group the output and log files generated by the script. 
I will rename it.

File: src/gt_merger/args.py [hunk 2]
```diff
@@ -28,20 +29,21 @@ def get_parser():
 
     parser.add_argument('--gtFile', type=str, required=True, help='Path to XLSX file including the Ground Truth data')
 
-    parser.add_argument('--dataDir', type=str, default='data',
-                        help='Folder including the sub-folder related to each GT Data files')
+    parser.add_argument('--outputDir', type=str, default=constants.OUTPUT_DIR,
+                        help='Path to folder directory used to save merged and log data generated after merge')
```
Comment (barbeau) [inline src/gt_merger/args.py]:
```suggestion
                        help='Path to directory where the merged data and log data will be output')
```

**Pull request merged**